Sources/Core/SoftwareBreakpointManager.cpp

```diff
@@ -83,4 +93,24 @@ ErrorCode SoftwareBreakpointManager::disableLocation(Site const &site) {
 
   return kSuccess;
 }
+
+void SoftwareBreakpointManager::enable(Target::Thread *thread) {
+  super::enable(thread);
+
+  _enabled = true;
+}
+
+void SoftwareBreakpointManager::disable(Target::Thread *thread) {
+  super::disable(thread);
```

Comment:
Do we have to pass `thread` here? Can't we just `DS2_ASSERT(thread == nullptr)` and call `supper::disable()`?